install_script.sh.template

```diff
@@ -1057,6 +1057,13 @@ if [ "$OS" == "RedHat" ]; then
                 exit;
             fi
         else
+            if [ ${#apm_libraries[@]} -gt 0 ]; then
```

Comment:
Shouldn't this outside of the innermost `if/else` (so on line 1052)?
This code is only reached when the user doesn't specify a minor vesion (ie. attempts to install the latest), while I think you'd want to error out when the user attempt to install anything through the installer on CentOS 6 and equivalent

Reply:
Oh yes actually. Thank you